GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
Is this method really necessary as it is the GVRActivity that instantiates the View Manager. The context in essence should always be available to the activity from an applications point of view after onInit().